docs/modules/ROOT/pages/tutorials/tutorial.adoc

```diff
@@ -76,7 +74,12 @@ IMPORTANT: On some laptops, running Minikube on battery power severely undermine
 * `kubectl apply -k wordpress`
 
 . Install the CRDs K8up uses:
-* `kubectl apply -f \https://github.com/k8up-io/k8up/releases/download/{k8up_version}/k8up-crd.yaml`
+ifeval::["{page-component-version}" == "master"]
+* `kubectl apply -f \https://github.com/k8up-io/k8up/releases/download/v{page-component-latest-version}.0/k8up-crd.yaml`
```

Comment:
This will always point to the x.0 patch. Doesn't the variable include that?

Reply:
Unfortunately no.